qemu/start-qemu

```diff
@@ -39,7 +39,7 @@ if [ "$QEMU_HDA" ]; then
 	fi
 	# http://wiki.qemu.org/download/qemu-doc.html#Invocation
 	#qemuArgs+=( -hda "$QEMU_HDA" )
-	qemuArgs+=( -drive file="$QEMU_HDA",index=0,media=disk,format=raw,discard=unmap )
+	qemuArgs+=( -drive file="$QEMU_HDA",index=0,media=disk,format=qcow2,discard=unmap )
```

Comment:
Do you think we should maybe instead just remove `format=...` entirely and let QEMU's auto-detection do its thing?